contracts/lib.rs

```diff
@@ -1080,17 +1105,22 @@ pub mod prosopo {
 
         /// Add an operator
         #[ink(message)]
-        pub fn add_prosopo_operator(&mut self, operator_account: AccountId) {
+        pub fn add_prosopo_operator(&mut self, operator_account: AccountId) -> Result<(), Error> {
```

Comment:
This function should be removed. It must've been added back in by a merge. Please delete it.

Reply:
fixed